### PR TITLE
ADS-342: add regression test for opera-rtc-s1-browse

### PR DESCRIPTION
**Pull request by asjohnston-asf**
Should be ready for review. I was able to run the test via workflow dispatch successfully against both Harmony UAT and Harmony production.

### CODE REVIEW

File: test/opera-rtc-s1-browse/opera-rtc-s1-browse_Regression.ipynb
Comment (owenlittlejohns) [inline test/opera-rtc-s1-browse/opera-rtc-s1-browse_Regression.ipynb:43]: This is probably the nicer way to make a request run across multiple environments rather than using different concept IDs for local/SIT/UAT versus production. Nice.

**Review by owenlittlejohns — APPROVED** [review]
I ran this locally against UAT and production. The production test passed, but the UAT test gave me a `Forbidden` error. Looks like my EDL user doesn't have access to the ASF test datapool, which makes sense.

The only other comment I'd have is that the notebook is maybe a little lacking in some description of what the test is doing, but as someone who is familiar with HyBIG, it does make sense to me. I guess ultimately it's up to you and other developers of this service how much you need prose around the test to explain what the notebook is doing.

**Comment by asjohnston-asf** [comment]
For UAT, you'll need to authorize application client id `LqWhtVpLmwaD4VqHeoN7ww` in EDL UAT

For production, you'll need to authorize application client id `BO_n7nTIlMljdvU6kRRB3g` in EDL Ops.

File: test/opera-rtc-s1-browse/opera-rtc-s1-browse_Regression.ipynb
Comment (asjohnston-asf) [reply test/opera-rtc-s1-browse/opera-rtc-s1-browse_Regression.ipynb:43]: Exactly. Collection short name and granule UR are almost always my preferred identifiers over concept IDs. Concept IDs are devoid of human-readable context, are generated by CMR, and differ from environment to environment. Short name and granule UR include human readable details, are assigned by the DAAC, and are (usually) the same from environment to environment.

**Comment by asjohnston-asf** [comment]
I'm content to leave the notebook as-is. If this PR can be merged I'll mark the work complete on my end.

**Comment by owenlittlejohns** [comment]
@asjohnston-asf - sounds good. Do you have a Jira ticket number I can stick on the squash-and-merge commit message?

Also, feel free to just merge this PR if you have sufficient privileges to do so. We opt for the squash-and-merge strategy to keep the commit history fairly clean.

**Comment by asjohnston-asf** [comment]
This is a component of the larger https://bugs.earthdata.nasa.gov/browse/ADS-342

I don't have write access to nasa/harmony-regression-tests, so someone else will have to merge on my behalf

**Pull request merged**
